.prettierrc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "semi": false,
+  "bracketSpacing": false,
+  "trailingComma": "all",
+  "singleQuote": true,
+  "printWidth": 70,
+  "useTabs": true
```

Comment:
useTabs can't be right... Projects should be using spaces...

It's disappointing that this config couldn't be read from the eslint config and / or xo...

Reply:
hmm... cause after i run `prettier --write generator.js` then `npm run lint` it would complain about tabs as such

```
  generator.js:10:1
  ×  10:1  Replace ·· with ↹        prettier/prettier
  ×  11:1  Replace ·· with ↹        prettier/prettier
  ×  12:1  Replace ···· with ↹↹     prettier/prettier
  ×  13:1  Replace ···· with ↹↹     prettier/prettier
  ×  14:1  Replace ·· with ↹        prettier/prettier
  ×  16:1  Replace ·· with ↹        prettier/prettier
  ×  17:1  Replace ·· with ↹        prettier/prettier
  ×  18:1  Replace ···· with ↹↹     prettier/prettier
  ×  19:1  Replace ···· with ↹↹     prettier/prettier
  ×  20:1  Replace ······ with ↹↹↹  prettier/prettier
  ×  21:1  Replace ···· with ↹↹     prettier/prettier
```

thus adding the `useTabs` seems to solve it.  There's no tabs option configured here though https://github.com/Tram-One/eslint-config-tram-one/blob/master/index.js maybe eslint uses tab by default?

Reply:
It looks like these should be set by default https://github.com/xojs/xo#prettier, I wonder why they need to be redefined here... 🤔